howso/direct/client.py

```diff
@@ -5074,6 +5078,85 @@ def set_auto_analyze_params(  # noqa: C901
             **kwargs
         )
         self._auto_persist_trainee(trainee_id)
+    
+    def get_auto_ablate_params(self, trainee_id: str):
```

Comment:
`get_auto_ablate_params` and `set_auto_ablate_params` need to be added to the howso/client/base.py